src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -486,6 +486,39 @@ public unsafe bool CursorGrabbed
             }
         }
 
+        /// <inheritdoc />
+        public int CurrentMonitorPpi
+        {
+            get
+            {
+                int widthInMm;
+                int heightInMm;
+                int widthInPixels;
+                int heightInPixels;
+                float horizontalPpi;
+                float verticalPpi;
+
+                unsafe
+                {
+                    GraphicsLibraryFramework.Monitor* monitor =
```

Comment:
Can `GraphicsLibraryFramework.Monitor` ever be null? If yes, consider check for null and either throw a meaningful exception or return a default Ppi/Dpi value here

Reply:
Would be cool to use more of the new C# Nullable stuff (for F# simply an option), then it would be easier to figure out if it can be null.

Reply:
or return a `NullMonitor` instead of `null`

Reply:
Issue should be resolved in the latest commit.